### PR TITLE
chore: update version and changelog for 4.0.2 release SDKCF-4000

### PR DESCRIPTION
**Pull request by apparition47**
# Description
4.0.2 prep

## Links
SDKCF-4000

# Checklist
- [x] I have read the [contributing guidelines](../CONTRIBUTING.md).
- [ ] I wrote/updated tests for new/changed code
- [x] I removed all sensitive data **before every commit**, including API endpoints and keys
- [ ] I ran `fastlane ci` without errors

### CODE REVIEW

**Review by donnie-jp — APPROVED** [review]
👍

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,11 @@
 ## Changelog
 
+### 4.0.2 (2021-07-21)
+- Improvements:
+	- Avoid displaying campaign message when image can't be downloaded [SDKCF-3977]
+	- Avoid displaying campaign message when display permission request was not successful [SDKCF-3976]
+	- Increase the clickable area of the IAM close button "x" [SDKCF-3958]
```
Comment (donnie-jp) [inline CHANGELOG.md]: We use past tense for changelogs (see the others below), like "Increased the...".

**Comment by apparition47** [comment]
Changed release date to `2021-07-21`

File: CHANGELOG.md (hunk 1 above)
Comment (apparition47) [reply CHANGELOG.md]: Oops, I knew that 😵

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,11 @@
 ## Changelog
 
+### 4.0.2 (2021-07-21)
+- Improvements:
+	- Changed campaigns to not display when attached image cannot be downloaded. [SDKCF-3977]
+	- Changed campaigns to not display upon display permission request failure. [SDKCF-3976]
+	- Increased clickable area of the IAM close button "x". [SDKCF-3958]
```
Comment (imaginaris) [inline CHANGELOG.md:7]: Just one more recent fix: SDKCF-4020
https://github.com/rakutentech/ios-inappmessaging/pull/102

Comment (apparition47) [reply CHANGELOG.md:7]: Oops sorry, didn't mean to leave out your work. Kept thinking that ticket was a SDKCF-3958 regression

Comment (imaginaris) [reply CHANGELOG.md:7]: No worries!

**Review by imaginaris — APPROVED** [review]

**Pull request merged**